src/codeu/chat/server/Model.java

```diff
@@ -680,4 +713,67 @@ public Uuid conversationID(Uuid message) {
     String query = "SELECT CONVERSATIONID FROM MESSAGES WHERE ID = ?;";
     return dbConnection.getConversationID(parameters, query);
   }
+
+  public void add(Uuid user) {
+    String query;
+    Vector<String> parameters = new Vector<>();
+
+    try {
+      parameters.add(SQLFormatter.sqlID(user));
+      query = "INSERT INTO USER_CLUSTER (ID, CLUSTER) " +
+          "VALUES ( ?, -1);";
+
+      dbConnection.dbUpdate(parameters, query);
+
+      LOG.info("User cluster added: " + user);
+    } catch (Exception e) {
+      LOG.info(
+          "newUserCluster fail - Verify connection and try again shortly");
+      System.err.println(e.getClass().getName() + ": " + e.getMessage());
+      System.exit(0);
+    }
+  }
+
+  public void update(Uuid user, int cluster) {
```

Comment:
Rename method to be a little more descriptive. Perhaps something like assignUserToCluster() ?